README.md

```diff
@@ -0,0 +1,49 @@
+# Afkanerd-OpenWebSockets
+
+## **Setup and Config**
+
+### **Global config**
+
+- Clone or download repository to local machine.
+
+  > `git clone https://github.com/Wisdom-Nji/Afkanerd-OpenWebSockets.git`
+
+- Install dependencies with npm.
+
+  > `npm install`
+
+- Grant execute permission to install script.
+
+  > `sudo chmod 777 install.sh`
+
```

Comment:
You make the script executable - rather than changing user read/write priviledge
`chmod a+x install.sh`